.nvmrc

```diff
@@ -1 +1 @@
-v16.20.2
+14.17.0
```

Comment:
What's the reason for requiring a deprecated version of Node here?

Reply:
This was a change that was made in #1545 and lost in a rebase – thanks for spotting! Reinstated in c243b73726ca76835fd1bc4837d6b51b34062154.